ui/src/app/components/runs/filters/number-range-filter/number-range-filter.component.ts

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright 2018 ABSA Group Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {AfterViewInit, Component, Input, OnInit} from '@angular/core';
+import {ClrDatagridFilterInterface} from "@clr/angular";
+import {DagRunModel} from "../../../../models/dagRuns/dagRun.model";
+import {Subject, Subscription} from "rxjs";
+import {debounceTime, distinctUntilChanged} from "rxjs/operators";
+
+@Component({
+  selector: 'app-number-range-filter',
+  templateUrl: './number-range-filter.component.html',
+  styleUrls: ['./number-range-filter.component.scss']
+})
+export class NumberRangeFilterComponent implements ClrDatagridFilterInterface<DagRunModel>, AfterViewInit {
+  @Input() removeFiltersSubject: Subject<any>;
+  @Input() property: string;
+  value: {from: number, to: number} = {from: undefined, to: undefined};
+
+  //clarity interface
+  changes: Subject<any> = new Subject<any>();
+  //angular
+  modelChanges: Subject<any> = new Subject<any>();
+  modelSubscription: Subscription;
+
+  constructor() { }
+
+  ngAfterViewInit(): void {
+    this.modelSubscription = this.modelChanges.pipe(
+      debounceTime(500),
+      distinctUntilChanged()
+    ).subscribe(newValue => {
+      this.changes.next();
+    });
+
+    this.removeFiltersSubject.subscribe(_ => this.onRemoveFilter());
+  }
+
+  accepts(value: DagRunModel): boolean {
```

Comment:
Maybe rename `value` to `model`. It could be slightly misleading because of the `this.value`

Reply:
renamed to 'item' as it is named in ClrDatagridFilterInterface